### PR TITLE
Add shared ESLint config

### LINKED ISSUE #12: Add shared ESLint config
Add the shared ESLint config to the project.

### PR DESCRIPTION
**Pull request by whymarrh**
Closes #12

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  env: {
+    'browser': true,
+  },
+
+  extends: [
+    '@metamask/eslint-config',
+  ],
+
+  globals: {
+    'web3': true,
```
Comment (Gudahtt) [inline .eslintrc.js]: Why is this here?

Comment (whymarrh) [reply .eslintrc.js]: Copy-paste. I'll remove it.
